### ISSUE TITLE
QGEP SWMM: Export subcatchments in [ha] rather than [m2]

### ISSUE THREAD

**Issue by tproduit** [QGEP/QGEP#697]
**subcatchments  area is exported in m2**
It should rather exported in [ha], swmm need [ha]

**Desktop :**
 - QGEP datamodel version 1.5.6

**Comment by sjib** [comment]
If swmm needs it in [ha] we should make the calculation from [m2] to [ha] in the swmm plugin and not change the QGEP datamodel, else we create other problems, as the unit is defined by the VSA-DSS model.

**Comment by tproduit** [comment]
My proposition is to change the corresponding view in this way (surface_area is already recorded in ha in QGEP):
```       
CASE
            WHEN ca.surface_area IS NULL THEN st_area(ca.perimeter_geometry)/10000
            WHEN ca.surface_area < 0.01 THEN st_area(ca.perimeter_geometry)/10000
            ELSE (ca.surface_area::numeric)::double precision
END AS area,
```

**Comment by TriformMoritz** [comment]
Yes, we had this problem, testing by changing the view like proposed seems to work.

**Comment by sjib** [comment]
@TriformMoritz Thanks for checking - I made a pull request to include this: https://github.com/QGEP/datamodel/pull/181
@tproduit Can you review?

**Comment by tproduit** [comment]
@sjib I reviewed it, it's ok to me.

**Comment by sjib** [comment]
For reference st_area calculates values in [m2] - see https://postgis.net/docs/ST_Area.html

**Comment by sjib** [comment]
Fixed with QGEP/datamodel#181
